tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -856,252 +809,262 @@ describe(__filename, () => {
 
     // See this other user profile page.
     const params = { userId: user.id };
-    const root = renderUserProfile({ params, store });
+    renderUserProfile({ params });
 
-    expect(root.find('.UserProfile-reviews')).toHaveLength(0);
+    expect(screen.queryByText('My reviews')).not.toBeInTheDocument();
+    expect(screen.queryByText('It is Okay')).not.toBeInTheDocument();
```

Comment:
I'm not sure we need this. Is there a case where "My reviews" would be absent but we'd still see an individual review?

Reply:
I doubt it, and not with our current component structure.